code/testsuite/base-xml.ftl

```diff
@@ -102,7 +102,7 @@
 		</skin>
 		<cr><@pcstring tag="CR"/></cr>
 		<face>
-		<face><@pcstring tag="FACE"/></face>
+			<face><#if pc.val.os_size!false == true><#if pc.val.face[0] == pc.val.face[1]>${pc.val.face[0]} ft.<#else>${pc.val.face[0]} ft. by test ${pc.val.face[1]} ft.</#if><#else>${pcstring('FACE')}</#if></face>
```

Comment:
Yes, that would help. Especially since I have discovered freemarker is extremely strict about values being absent.